litellm-proxy-extras/litellm_proxy_extras/schema.prisma

```diff
@@ -96,6 +96,7 @@ model LiteLLM_TeamTable {
     max_budget Float?
     spend      Float    @default(0.0)
     models     String[]
+    mcp_servers String[] @default([])
```

Comment:
for this PR can we just add the `LiteLLM_MCPServerTable`

it's not clear to me how we'll end up enforcing mcp / knowledge base permissions as yet. This might go under `permissions` in the tables. 

Would prefer to first just add the table to main

Reply:
I was going to maintain the list of mcp server ids during the CRUD operations, similar to how `VerificationToken` table uses `models` and Team table uses both `models` and `members`. 

I was thinking there was going to be the special `all-team-servers` and `all-proxy-servers` to follow the same practice for model behavior

Reply:
Is it possible to follow a more extensible design? 

create a new table called ObjectPermissions, this table has a column called mcp_servers

(in the future when we need knowledge base permissions we can add to this table)

Reply:
Done

Reply:
There is no connection to the table and the respective `VerificationToken` and `Team` tables without the entry or relation to this new table. Is that what you are going for?

```prisma
model LiteLLM_ObjectPermissionTable {
  permission_id  String         @id @default(uuid())
  mcp_servers    String[]       @default([])
}

...

model LiteLLM_VerificationToken {
  ....
  object_permission LiteLLM_ObjectPermissionTable?   @relation(fields: [permission_id], references: [permission_id])
}
```

Reply:
yup exactly !